packages/platforms/react-native/lib/NativeBugsnagPerformance.ts

```diff
@@ -1,6 +1,8 @@
 import type { TurboModule } from 'react-native/Libraries/TurboModule/RCTExport'
 import { TurboModuleRegistry } from 'react-native'
 
+export type Encoding = 'utf8' | 'base64'
```

Comment:
I think this is what's causing the `Union types are unsupported in structs` codegen error.

We can use this type on the JS side, but in the turbo module spec this should just be a `string`